OsmAnd/src/net/osmand/plus/helpers/FontCache.java

```diff
@@ -0,0 +1,40 @@
+package net.osmand.plus.helpers;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.util.Log;
+
+public class FontCache {
```

Comment:
Recently we removed custom fonts due to [this issue](https://github.com/osmandapp/OsmAnd/issues/19123), better use default fonts